core/node/node_framework/examples/main_node.rs

```diff
@@ -356,47 +354,6 @@ impl MainNodeBuilder {
         Ok(self)
     }
 
-    fn add_consensus_layer(mut self) -> anyhow::Result<Self> {
```

Comment:
This example is incomplete, and since the configs are loaded differently here, I decided to remove it instead of reworking.
Probably once we migrate the main node to the framework fully, this example is to be removed anyways.